components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.h

```diff
@@ -60,10 +60,17 @@
 @property(nonatomic, assign) BOOL addLeadingPadding;
 
 /**
- The inset or outset margins for the rectangle surrounding all of the button’s content.
+ The inset or outset margins for the rectangle surrounding all of the cell's content.
 
  Defaults to @c UIEdgeInsetsZero.
  */
 @property(nonatomic, assign) UIEdgeInsets contentEdgeInsets;
 
+/**
+ The inset or outset margins for the rectangle surrounding all of the cell's content.
+
+ Defaults to {0, 0, 0, -32}.
```

Comment:
Shouldn't this be `{-16,0,0,0}`?

Reply:
Good catch, it should be the documented value!

Reply:
Actually, it should be a combination of the two. 😨

Reply:
Done.